smart_open/ssh.py

```diff
@@ -38,6 +41,36 @@
 
 DEFAULT_PORT = 22
 
+URI_EXAMPLES = (
+    '[ssh|scp|sftp]://username@host//path/file',
```

Comment:
comparing to other URI examples looks strange (not sure than all understand `[X|Y|Z]` form)